### PR TITLE
fixed (default) browser support

### PR DESCRIPTION
**Pull request by donnell794**
changed opening url to a system call for better support
fixes: #80 
fixes: #12

### CODE REVIEW

File: socli/socli.py [hunk 1]
```diff
@@ -456,9 +456,15 @@ def keypress(self, size, key):
             elif key in {'up', 'n', 'N'}:
                 self.answer_text.prev_ans()
             elif key in {'o', 'O'}:
-                import webbrowser
                 HEADER.event('browser', "Opening in your browser...")
-                webbrowser.open(self.url)
+                if sys.platform == 'darwin':
+                    import subprocess
+                    subprocess.Popen(['open', self.url])
+                elif: sys.platform == 'win32':
```
Comment (gautamkrishnar) [inline socli/socli.py]: This line has a syntax error, Please fix it. The correct syntax is:
```python
elif sys.platform == 'win32':
```

**Comment by donnell794** [comment]
fixed

**Comment by gautamkrishnar** [comment]
We can merge this when I release the next version of the app... There's a lots of work need in the develop branch...

**Comment by gautamkrishnar** [comment]
@donnell794 Merged 👍  I will refractor on next release... Thanks for your contribution ❤️

**Comment by gautamkrishnar** [comment]
@donnell794 Is socli now availbale on macports? i am not a mac user ... 😄

**Comment by donnell794** [comment]
@gautamkrishnar not yet, working on MacPorts and Homebrew installation

**Comment by gautamkrishnar** [comment]
@donnell794  okay please inform me when it is available

**Pull request merged**